docs/SPEC.md

```diff
@@ -1723,6 +1723,34 @@ from(bucket: "telegraf/autogen")
 	|> median()
 ```
 
+##### Moving Average
+
+Moving Average is an aggregate operation.
+For each aggregated column, it means the values of the following records for a defined window range.
+Moving Average is defined as
+```
+movingAverage = (every, period, column="_value", timeSrc="_stop",timeDst="_time", tables=<-) =>
```

Comment:
Are the `timeSrc` and `timeDst` parameters necessary? By default the output time column is `_stop` for all aggregates, and since the aggregate, in this case mean, is encapsulated within the function declaration, maybe these parameters shouldn't be configurable.

Also the `column` parameter is not used in the `mean()` call.